### DISCUSSION TITLE
关于 AI Summary 的缺点五

### DISCUSSION THREAD

**Discussion by wpv-chan** [thinkingjimmy/Learning-Prompt#48] [Ideas]
我个人的理解重点其实是人为定义的，对同一份内容，不同需求的阅读者就会有不同的重点倾向，如果在summary后面加上想关注的主题我觉得效果还是不错的，下面是我测试的一个例子：

Question: Summarize the papers, focus on their advances

The papers discussed in Sakpere2017 and Kunhoth2020 focus on advances in indoor positioning and wayfinding systems. Sakpere2017 provides a state-of-the-art survey of indoor positioning and navigation systems, discussing the strengths and limitations of various positioning techniques and technologies and proposing possible research directions for further improvement. The studies discussed in Sakpere2017 pages 21-22 focus on improving indoor positioning and tracking systems using magnetic fields, with advances in excitation of magnetic sources, sensor output, and sensor positions. Kunhoth2020 discusses three different indoor positioning and wayfinding systems, including advances in keyframe extraction, topological map creation, localization, routing, barcode-based navigation, and wearable virtual ushers. Other papers discussed in Kunhoth2020 pages 19-20 and Kunhoth2020 pages 37-37 propose advances in building distinct models for different indoor environments, efficient signal processing, and improved accuracy in challenging environments.

Question: Summarize the papers, focus on related works

There are several papers related to indoor positioning and wayfinding systems. WLAN fingerprinting methods for indoor positioning have been proposed by Kaemarungsi (2005), Jiang (2012), and Niu et al. (2013) with varying degrees of accuracy and energy efficiency (Sakpere2017 pages 30-31). Computer vision-based line segment matching and trifocal tensor have been proposed for indoor navigation, as well as detecting doors in indoor areas using a miniature camera and a computer with a door detection algorithm (Kunhoth2020 pages 10-11). Other related works include foot-mounted inertial systems, substitute vision systems, and GPS-enabled mobile phones, as well as feature detection and matching, deep learning, and wearable RGBD indoor navigation systems (Kunhoth2020 pages 35-36). Additionally, there are papers on object recognition, scene labeling, and navigation systems for the visually impaired (Kunhoth2020 pages 37-37). The papers are summarized briefly and include direct links to their respective sources (Kunhoth2020 pages 37-37).

References

1. (Sakpere2017): Sakpere, Wilson, Michael Adeyeye-Oshin, and Nhlanhla B.W. Mlitwa. "A state-of-the-art survey of indoor positioning and navigation systems and technologies." South African Computer Journal 29.3 (2017): 145-197. https://doi.org/10.18489/sacj.v29i3.452.

2. (Kunhoth2020): Kunhoth, Jayakanth, et al. "Indoor positioning and wayfinding systems: a survey." Journal of Ambient Intelligence and Humanized Computing, vol. 11, no. 5, 2020, pp. 2145-2167.

**Comment by thinkingjimmy** [comment]
感谢分享。

是的，其实文章里我也提到过这个方法，如果我能将 hightlight 数据给 AI，那样结果会更好。

但这里有个鸡生蛋、蛋生鸡的问题。我没看过这个 paper 我如何知道它是否包含我关注的内容呢？

可能 paper 还好点，主题明确。

但如果是我文章里提到的 podcast 呢？内容很宽泛，在我没听 podcast 前，我根本不知道他们聊的内容跟我感兴趣的内容有关，甚至有些是他们聊了，我才知道我感兴趣。比如文章里提到的「为何印度永远成为不了中国」这个话题，我没听，我是不知道他们会讲这个的，我在听之前，我都没想过这个问题。那我要如何将这些信息输入给 AI 呢？
